ndc_lib/src/interpreter/evaluate/index.rs

```diff
@@ -63,7 +64,7 @@ impl EvaluatedIndex {
 
 pub(crate) fn evaluate_as_index(
     expression_location: &ExpressionLocation,
-    environment: &mut EnvironmentRef,
+    environment: &Rc<RefCell<Environment>>,
```

Comment:
type is het zelfde, maar `EnvironmentRef` was een alias (ik vind dat minder leesbaar)